lib/kitchen/driver/dokken.rb

```diff
@@ -149,10 +151,14 @@ def build_work_image(state)
       end
 
       def work_image_dockerfile
-        from = "FROM #{platform_image}"
-        custom = ['RUN /bin/sh -c "echo Built with Test Kitchen"']
-        Array(config[:intermediate_instructions]).each { |c| custom << c }
-        [from, custom].join("\n")
+        dockerfile_contents = [
+          "FROM #{platform_image}",
+          'LABEL X-Built-By=kitchen-dokken'
```

Comment:
Should not assume `/bin/sh` availability. Replace with a `LABEL` Dockerfile directive.